react/features/toolbox/reducer.js

```diff
@@ -207,6 +207,14 @@ function _setButton(state, { button, buttonName }): Object {
         ...button
     };
 
+    // In filmstrip-only mode we only show buttons if they're film-strip only
+    // enabled, so we don't need to update if this isn't the case.
+    if (interfaceConfig.filmStripOnly && !selectedButton.filmStripOnlyEnabled) {
```

Comment:
Will there be a problem if the update happens though? I'm asking because a reducer should be a pure function of the current state and the action it reduces while interfaceConfig is something else.